### DISCUSSION TITLE
Latex render bug with '$Z_{{1 - \beta}}$ '

### DISCUSSION THREAD

**Discussion by VISTALL** [requarks/wiki#3874] [Error / Bug Report]
Hello, today I found render issue with latex inside markdown

Page text

```latex
$Z_{{1 - \beta}}$ 
```

Version - last active.

Page preview (and expected result)

![image](https://user-images.githubusercontent.com/542934/116118514-0f939e00-a6c6-11eb-8533-f72cc3b520f9.png)

Actual: 

empty wiki page

![image](https://user-images.githubusercontent.com/542934/116118863-6d27ea80-a6c6-11eb-8f44-6710d83202df.png)

Thanks

**Comment by VISTALL** [comment]
Also if make text like

```latex
$Z_{1 - \beta}$ 
```

preview - same.

Wiki page will not broken, but result is not equal preview

![image](https://user-images.githubusercontent.com/542934/116119194-d27bdb80-a6c6-11eb-99de-71dc2e1f1321.png)

**Comment by BPowell76** [comment]
I have also noticed this issue. For me, listing out latex equations inside a Markdown Table causes issues if the equation ends with a `}$`. It won't render properly and produces what @VISTALL shared above. To work around this, I can remove the {} for single-element items (i.e., use `$e^x$` instead of `$e^{x}$`). For more complex equations, this won't work.
